cmd/beacon-chain/flags/base.go

```diff
@@ -100,30 +100,33 @@ var (
 		Usage: "Comma-separated list of API module names. Possible values: `" + PrysmAPIModule + `,` + EthAPIModule + "`.",
 		Value: PrysmAPIModule + `,` + EthAPIModule,
 	}
-	// DisableGRPCGateway for JSON-HTTP requests to the beacon node.
-	DisableGRPCGateway = &cli.BoolFlag{
+	// Deprecated: DeprecatedDisableGRPCGateway for JSON-HTTP requests to the beacon node.
+	DeprecatedDisableGRPCGateway = &cli.BoolFlag{
```

Comment:
It's not used anywhere, there isn't a disable gateway anyways, wonder if I should deprecate or just remove

Reply:
Since we have the flag could we use it to disableGRPC instead so that we can use it to toggle between HTTP and GRPC when we are finally testing it's replacement?

Reply:
will move this out and into feature